.cci.jenkinsfile

```diff
@@ -1,17 +1,23 @@
 // Documentation: https://github.com/coreos/coreos-ci/blob/master/README-upstream-ci.md
 
-cosaPod(buildroot: true) {
+cosaPod {
     checkout scm
 
-    // hack to satisfy golang compiler wanting to cache things
-    shwrap("mkdir cache")
-    withEnv(["XDG_CACHE_HOME=${env.WORKSPACE}/cache"]) {
-        // first, run gofmt/govet/unit tests
-        stage("Unit Tests") {
-            shwrap("./test")
+    buildPod {
+        // hack to satisfy golang compiler wanting to cache things
+        shwrap("mkdir cache")
+        withEnv(["XDG_CACHE_HOME=${env.WORKSPACE}/cache"]) {
+            stage("Build") {
+                shwrap("make")
+                shwrap("make install DESTDIR=install")
```

Comment:
The `DESTDIR=install` bit and `overlays: "install"` bit below is kinda awkward. We can look at adding more sugar for this in the future.

Reply:
Yeah definitely, centralizing and standardizing this is IMO a lot of the goal.